### PR TITLE
docs: upgrade from 2.1 to 2.2

### PR DESCRIPTION
**Pull request by STRRL**
Signed-off-by: STRRL <im@strrl.dev>

close https://github.com/chaos-mesh/website/issues/281

### CODE REVIEW

**Comment by STRRL** [comment]
/cc @YangKeao @iguoyr @cwen0 @Oreoxmt 

PTAL

**Review by YangKeao — APPROVED** [review]
LGTM

File: i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+title: 从 2.1 升级到 2.2
+---
+
+在 2.2.0 版本中，Helm Charts 做了一些改变。这个文档将帮助你从 2.1.x 升级到 2.2.0。
+
+## 使用 Helm 升级
+
+### 第 1 步: 添加/更新 Chaos Mesh Helm 仓库
+
+添加 Chaos Mesh 到 Helm 仓库并更新:
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md:11]:
```suggestion
添加 Chaos Mesh 到 Helm 仓库并更新：
```

File: i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+---
+title: 从 2.1 升级到 2.2
+---
+
+在 2.2.0 版本中，Helm Charts 做了一些改变。这个文档将帮助你从 2.1.x 升级到 2.2.0。
+
+## 使用 Helm 升级
+
+### 第 1 步: 添加/更新 Chaos Mesh Helm 仓库
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md:9]:
```suggestion
### 第 1 步：添加/更新 Chaos Mesh Helm 仓库
```

File: i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+添加 Chaos Mesh 到 Helm 仓库并更新:
+
+```bash
+helm repo add chaos-mesh https://charts.chaos-mesh.org
+helm repo update
+```
+
+### 第 2 步: 迁移 `values.yaml` 文件
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md:18]:
```suggestion
### 第 2 步：迁移 `values.yaml` 文件
```

File: i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+这里是 Helm Chart 改变的列表:
+
+- 新配置: `chaosDaemon.mtls.enabled` 表示在 chaos-controller-manager 和 chaos-daemon 之间使用 mtls.
+- 新配置: `webhook.caBundlePEM` 表示用于 webhook 服务的 CA bundle.
+- 改变的值: `dashboard.serviceAccount` 从 `chaos-controller-manager` 改为 `chaos-dashboard`.
+- 改变的值: `webhook.FailurePolicy` 从 `Ignore` 改为 `Fail`.
+
+:::note 你可以从这里查看 Helm Chart values 详细的描述: https://github.com/chaos-mesh/chaos-mesh/blob/v2.2.0/helm/chaos-mesh/README.md :::
+
+### 第 3 步: 更新 CRD
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md:39]:
```suggestion
### 第 3 步：更新 CRD
```

File: i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+如果你的 Kubernetes 版本 <= 1.15, 你可以使用以下命令更新 Chaos Mesh CRD:
+
+```bash
+kubectl replace -f https://mirrors.chaos-mesh.org/v2.2.0/crd-v1beta1.yaml
+```
+
+:::note Chaos Mesh 2.2.x 将会是支持 kubernetes <1.19 的最后一系列版本. :::
+
+### 第 4 步: 使用 `helm upgrade` 升级 Chaos Mesh
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md:55]:
```suggestion
### 第 4 步：使用 `helm upgrade` 升级 Chaos Mesh
```

File: i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+```bash
+helm repo add chaos-mesh https://charts.chaos-mesh.org
+helm repo update
+```
+
+### 第 2 步: 迁移 `values.yaml` 文件
+
+如果你在安装 Chaos Mesh 的时候使用了一个特定的 `values.yaml`，我们建议你应用你的自定义配置到 Chaos Mesh 2.2.0 的 `values.yaml`。
+
+你可以通过这个命令获取默认的 `values.yaml`:
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md:22]:
```suggestion
你可以通过这个命令获取默认的 `values.yaml`：
```

File: i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+```bash
+helm show values chaos-mesh/chaos-mesh --version 2.2.0 > values.yaml
+```
+
+如果你不熟悉这些改变过的配置, 你可能不会依赖这些特定的功能, 忽略他们通常是安全的.
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md:28]:
```suggestion
如果你不熟悉这些改变过的配置，你可能不会依赖这些特定的功能，忽略他们通常是安全的。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+这里是 Helm Chart 改变的列表:
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md:30]:
```suggestion
这里是 Helm Chart 改变的列表：
```

File: i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+kubectl replace -f https://mirrors.chaos-mesh.org/v2.2.0/crd.yaml
+```
+
+如果你的 Kubernetes 版本 <= 1.15, 你可以使用以下命令更新 Chaos Mesh CRD:
+
+```bash
+kubectl replace -f https://mirrors.chaos-mesh.org/v2.2.0/crd-v1beta1.yaml
+```
+
+:::note Chaos Mesh 2.2.x 将会是支持 kubernetes <1.19 的最后一系列版本. :::
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md:53]:
```suggestion
:::note 
Chaos Mesh 2.2.x 将会是支持 Kubernetes < 1.19 的最后一系列版本。
:::
```

File: i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+
+### 第 4 步: 使用 `helm upgrade` 升级 Chaos Mesh
+
+你可以使用以下命令来升级 Chaos Mesh 到 2.2.0:
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md:57]:
```suggestion
你可以使用以下命令来升级 Chaos Mesh 到 2.2.0：
```

File: i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+
+### 第 3 步: 更新 CRD
+
+如果你的 Kubernetes 版本 >= 1.16, 你可以使用以下命令更新 Chaos Mesh CRD:
+
+```bash
+kubectl replace -f https://mirrors.chaos-mesh.org/v2.2.0/crd.yaml
+```
+
+如果你的 Kubernetes 版本 <= 1.15, 你可以使用以下命令更新 Chaos Mesh CRD:
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md:47]:
```suggestion
如果你的 Kubernetes 版本 <= 1.15，你可以使用以下命令更新 Chaos Mesh CRD：
```

File: i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+如果你不熟悉这些改变过的配置, 你可能不会依赖这些特定的功能, 忽略他们通常是安全的.
+
+这里是 Helm Chart 改变的列表:
+
+- 新配置: `chaosDaemon.mtls.enabled` 表示在 chaos-controller-manager 和 chaos-daemon 之间使用 mtls.
+- 新配置: `webhook.caBundlePEM` 表示用于 webhook 服务的 CA bundle.
+- 改变的值: `dashboard.serviceAccount` 从 `chaos-controller-manager` 改为 `chaos-dashboard`.
+- 改变的值: `webhook.FailurePolicy` 从 `Ignore` 改为 `Fail`.
+
+:::note 你可以从这里查看 Helm Chart values 详细的描述: https://github.com/chaos-mesh/chaos-mesh/blob/v2.2.0/helm/chaos-mesh/README.md :::
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md:37]:
```suggestion
:::note 
你可以在 [README](https://github.com/chaos-mesh/chaos-mesh/blob/v2.2.0/helm/chaos-mesh/README.md) 查看 Helm Chart 配置的详细描述。
:::
```

File: i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
+
+### 第 3 步: 更新 CRD
+
+如果你的 Kubernetes 版本 >= 1.16, 你可以使用以下命令更新 Chaos Mesh CRD:
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md:41]:
```suggestion
如果你的 Kubernetes 版本 >= 1.16，你可以使用以下命令更新 Chaos Mesh CRD：
```

File: i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md [hunk 14]
```diff
@@ -0,0 +1,65 @@
...
+
+你可以使用以下命令来升级 Chaos Mesh 到 2.2.0:
+
+```bash
+helm upgrade <release-name> chaos-mesh/chaos-mesh --namespace=<namespace> --version=2.2.0 <--other-required-flags>
+```
+
+## 询问社区
+
+如果你对升级 Chaos Mesh 有任何问题，请在 [Slack Channel](https://cloud-native.slack.com/archives/C0193VAV272)，GitHub [Issues](https://github.com/chaos-mesh/chaos-mesh/issues/new?assignees=&labels=&template=question.md) 和 [Discussions](https://github.com/chaos-mesh/chaos-mesh/discussions/new) 联系我们.
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md:65]:
```suggestion
如果你对升级 Chaos Mesh 有任何问题，请在 [Slack Channel](https://cloud-native.slack.com/archives/C0193VAV272)，GitHub [Issues](https://github.com/chaos-mesh/chaos-mesh/issues/new?assignees=&labels=&template=question.md) 和 [Discussions](https://github.com/chaos-mesh/chaos-mesh/discussions/new) 联系我们。
```

**Comment by STRRL** [comment]
Updated! PTAL again ❤️

@Oreoxmt

File: docs/upgrade-from-2.1-to-2.2.md [hunk 15]
```diff
@@ -0,0 +1,65 @@
...
+
+```bash
+kubectl replace -f https://mirrors.chaos-mesh.org/v2.2.0/crd-v1beta1.yaml
+```
+
+:::note Chaos Mesh 2.2.x would be the last series of releases that support kubernetes < 1.19. :::
+
+### Step 4: Upgrade Chaos Mesh by `helm upgrade`
+
+Then you could execute following command to upgrade Chaos Mesh to the 2.2.0:
```
Comment (Oreoxmt) [inline docs/upgrade-from-2.1-to-2.2.md:57]:
```suggestion
Then you can execute the following command to upgrade Chaos Mesh to the 2.2.0:
```

File: docs/upgrade-from-2.1-to-2.2.md [hunk 16]
```diff
@@ -0,0 +1,65 @@
...
+
+```bash
+helm upgrade <release-name> chaos-mesh/chaos-mesh --namespace=<namespace> --version=2.2.0 <--other-required-flags>
+```
+
+## Ask the Community
+
+If you have any question about upgrading Chaos Mesh, feel free to contact us at [Slack Channel](https://cloud-native.slack.com/archives/C0193VAV272), GitHub [Issues](https://github.com/chaos-mesh/chaos-mesh/issues/new?assignees=&labels=&template=question.md) and [Discussions](https://github.com/chaos-mesh/chaos-mesh/discussions/new).
```
Comment (Oreoxmt) [inline docs/upgrade-from-2.1-to-2.2.md:65]:
```suggestion
If you have any questions about upgrading Chaos Mesh, feel free to contact us at [Slack Channel](https://cloud-native.slack.com/archives/C0193VAV272), GitHub [Issues](https://github.com/chaos-mesh/chaos-mesh/issues/new?assignees=&labels=&template=question.md) and [Discussions](https://github.com/chaos-mesh/chaos-mesh/discussions/new).
```

File: docs/upgrade-from-2.1-to-2.2.md [hunk 17]
```diff
@@ -0,0 +1,65 @@
...
+```bash
+helm repo add chaos-mesh https://charts.chaos-mesh.org
+helm repo update
+```
+
+### Step 2: Migrate the `values.yaml` file
+
+If you installed Chaos Mesh with a certain `values.yaml`, we suggest you apply your customized configuration on the `values.yaml` of Chaos Mesh 2.2.0.
+
+You could fetch the default `values.yaml` by:
```
Comment (Oreoxmt) [inline docs/upgrade-from-2.1-to-2.2.md:22]:
```suggestion
You can fetch the default `values.yaml` by the following command:
```

File: docs/upgrade-from-2.1-to-2.2.md [hunk 18]
```diff
@@ -0,0 +1,65 @@
...
+
+```bash
+helm show values chaos-mesh/chaos-mesh --version 2.2.0 > values.yaml
+```
+
+If you do not recognize the changed configurations, you might not rely on that specific feature, it's usually safe to ignore them.
```
Comment (Oreoxmt) [inline docs/upgrade-from-2.1-to-2.2.md:28]:
```suggestion
If you do not recognize the changed configurations, you might not rely on that specific feature, it is usually safe to ignore them.
```

File: i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md [hunk 19]
```diff
@@ -0,0 +1,69 @@
...
+```
+
+如果你不熟悉这些改变过的配置，你可能不会依赖这些特定的功能，忽略他们通常是安全的。
+
+这里是 Helm Chart 改变的列表：
+
+- 新配置: `chaosDaemon.mtls.enabled` 表示在 chaos-controller-manager 和 chaos-daemon 之间使用 mtls.
+- 新配置: `webhook.caBundlePEM` 表示用于 webhook 服务的 CA bundle.
+- 改变的值: `dashboard.serviceAccount` 从 `chaos-controller-manager` 改为 `chaos-dashboard`.
+- 改变的值: `webhook.FailurePolicy` 从 `Ignore` 改为 `Fail`.
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md:35]:
```suggestion
- 新配置：`chaosDaemon.mtls.enabled` 表示在 `chaos-controller-manager` 和 `chaos-daemon` 之间使用 mtls。
- 新配置：`webhook.caBundlePEM` 表示用于 webhook 服务的 CA bundle。
- 改变的值：`dashboard.serviceAccount` 从 `chaos-controller-manager` 改为 `chaos-dashboard`。
- 改变的值：`webhook.FailurePolicy` 从 `Ignore` 改为 `Fail`。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md [hunk 20]
```diff
@@ -0,0 +1,69 @@
...
+如果你不熟悉这些改变过的配置，你可能不会依赖这些特定的功能，忽略他们通常是安全的。
+
+这里是 Helm Chart 改变的列表：
+
+- 新配置：`chaosDaemon.mtls.enabled` 表示在 `chaos-controller-manager` 和 `chaos-daemon` 之间使用 mtls。
+- 新配置：`webhook.caBundlePEM` 表示用于 webhook 服务的 CA bundle。
+- 改变的值：`dashboard.serviceAccount` 从 `chaos-controller-manager` 改为 `chaos-dashboard`。
+- 改变的值：`webhook.FailurePolicy` 从 `Ignore` 改为 `Fail`。
+
+:::note
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md:37]:
```suggestion
:::note 注意
```

File: i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md [hunk 21]
```diff
@@ -0,0 +1,69 @@
...
+kubectl replace -f https://mirrors.chaos-mesh.org/v2.2.0/crd.yaml
+```
+
+如果你的 Kubernetes 版本 <= 1.15，你可以使用以下命令更新 Chaos Mesh CRD：
+
+```bash
+kubectl replace -f https://mirrors.chaos-mesh.org/v2.2.0/crd-v1beta1.yaml
+```
+
+:::note
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/upgrade-from-2.1-to-2.2.md:55]:
```suggestion
:::note 注意
```

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**